tutorials/script/core/10_multiple_fallback_nodes.py

```diff
@@ -0,0 +1,127 @@
+                ("information_flow", "response_node"): cnd.exact_match(Message(text="info")),
+                ("fallback_node"): cnd.true()
+            }
+        },
+
+        "fallback_node": {
+            RESPONSE: Message(text="Unknown Command. Try again."),
+            TRANSITIONS: {"say_hi_node": cnd.exact_match(Message(text="hi"))
+            }
+        }
+    },
+    "authorization_flow": {
+        "response_node": {
+            RESPONSE: Message(text="Write your name, please."),
+            TRANSITIONS: {
+                ("authorization_flow", "auth_success_node"):
+                    cnd.exact_match(Message(text="admin")),
+                ("authorization_fallback_node"):
+                    cnd.true()
+            }
+        },
+
+        "auth_success_node": {
+            RESPONSE: Message(text="Authentication successful."),
+            TRANSITIONS: {
+                ("authorization_flow", "response_node"):
+                    cnd.exact_match(Message(text=""))
+            }
+        },
+
+        "authorization_fallback_node": {
+            RESPONSE: Message(text="User not found."),
+            TRANSITIONS: {
+                ("authorization_flow", "response_node"):
+                    cnd.exact_match(Message(text=""))
+            }
+        }
+    },
+
+    "information_flow": {
+        "response_node": {
+            RESPONSE: Message(text="What information you would like to know?"),
+            TRANSITIONS: {
+                ("information_flow", "weather_node"):
+                    cnd.exact_match(Message(text="weather")),
+                ("information_flow", "time_node"):
+                    cnd.exact_match(Message(text="time")),
+                ("information_fallback_node"):
+                    cnd.true()
+            }
+        },
+        "weather_node": {
+            RESPONSE: Message(text="It's sunny now!"),
+            TRANSITIONS: {
+                ("information_flow", "response_node"):
+                    cnd.true()
+            }
+        },
+        "time_node": {
+            RESPONSE: Message(text="It's tea time!"),
+            TRANSITIONS: {
+                ("information_flow", "response_node"):
+                    cnd.true()
+            }
+        },
+        "information_fallback_node": {
+            RESPONSE: Message(text="I don't know that information!"),
+            TRANSITIONS: {
+                ("information_flow", "response_node"):
+                    cnd.true()
+            }
+        }
+    }
+}
+
+# %% [markdown]
+"""
+As you can see, we've created specific fallback node for each individual flow (e.g. `authorization_fallback_node` for `authorization_flow`). We define transition to them in response node of the flow (actually it can be defined in any node in the flow) using `("information_fallback_node"): cnd.true()`. Due to the low priority this condition will trigger automatically if no other condition was triggered.
```

Comment:
Instead of defining that transition in every node, it should be defined as a `LOCAL` node transition.